tendermint-rs/Cargo.toml

```diff
@@ -43,7 +43,7 @@ signatory-dalek = { version = "0.10", optional = true }
 sha2 = { version = "0.8", optional = true, default-features = false }
 subtle-encoding = { version = "0.3", features = ["bech32-preview"] }
 tai64 = { version = "1", optional = true, features = ["chrono"] }
-x25519-dalek = { version = "0.3", optional = true, default-features = false, features = ["u64_backend"] }
+x25519-dalek = { version = "0.4.4", optional = true, default-features = false, features = ["u64_backend"] }
```

Comment:
One question: couldn't this simply be 0.4?

Reply:
I could make it "^0.4"?

Reply:
AFAIR, the `^` doesn't have any impact as `0.4` will be interpreted as `^0.4` b y default
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio
And if understand this correctly: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements
... `0.4.4` is equivalent to `^0.4.4` which means >= 0.4.4 and > 0.5. We probably want this. So it's fine to keep it as is. Sorry for the noise. I wasn't remembering the semantics here correctly.